web/Gruntfile.js

```diff
@@ -40,8 +45,12 @@ module.exports = function (grunt) {
                     '<%= config.srcScriptsDir %>/**/*.es.js'
                 ],
                 tasks: [
-                    'compilejs'
+                    'build'
                 ]
+            },
+            ngtemplates: {
+                files: ['app/templates/**/*.html'],
+                tasks: ['build']
```

Comment:
The watcher should really only change what is necessary, so compiling js for src js changes, and running `ngtemplates` when html templates change.